cmd/loop/quote.go

```diff
@@ -12,12 +12,25 @@ var quoteCommand = cli.Command{
 	Usage:       "get a quote for the cost of a swap",
 	ArgsUsage:   "amt",
 	Description: "Allows to determine the cost of a swap up front",
-	Action:      quote,
+	Flags: []cli.Flag{
+		cli.Uint64Flag{
+			Name: "conf-target",
```

Comment:
Same comment here re underscore.

Reply:
Current "inherited" style from `lncli` is underscore delimiters.

Reply:
Done.